src/routes/contract.ts

```diff
@@ -184,7 +237,8 @@ export async function contractRecordHandler(ctx: KoaContext) {
       contractTxId: record.contractTxId,
       warp,
       logger,
-      // don't set evaluation options for sub contracts - they'll be pulled on load
+      // we cannot use sort key as it is not applicable to sub contract
```

Comment:
Can you explain what you mean by this? All contract interactions have sort keys, no?

Reply:
here we read the sub contract to parse out the owner. a sortKey provided for the top level contract would not apply for the sub-contract we are trying to read (e.g. ANT's)